logical/pom.xml

```diff
@@ -85,14 +85,12 @@
     </dependency>
     
     <dependency>
-      <groupId>com.codahale.metrics</groupId>
+      <groupId>io.dropwizard.metrics</groupId>
```

Comment:
Is it used in logical?

Reply:
No, it is. Thanks for catching this. 
Also I have noticed that `java-exec` used `io.dropwizard.metrics` as transitive from `drill-common`.
For consistency I have removed  `io.dropwizard.metrics` from `drill-memory-base` (it can leverage metrics from `drill-common` too).

Reply:
It is not a best practice to rely on a transitive dependency for a compile scope dependency. I'd recommend specifying the dependency on `io.dropwizard.metrics` explicitly in case `java-exec` or `drill-memory-base` uses it.